docs/setup/independent/high-availability.md

```diff
@@ -0,0 +1,480 @@
+---
+approvers:
+- mikedanese
+- luxas
+- errordeveloper
+- jbeda
+title: Creating HA clusters with kubeadm
+---
+
+{% capture overview %}
+
+This guide will instruct you on how to install and set up a highly available Kubernetes cluster using kubeadm.
+
+* The goal of this document is to cover the areas that kubeadm was not designed to cover: the provisioning of hardware, configuration of multiple systems, and load balancing. The user is expected to handle this themselves with the aid of our documentation. 
+* Please note this guide is one potential solution - there are many ways to configure HA! If a better solution works for you, please use it. If you feel it can be adopted by the community, feel free to contribute it back.
+
+{% endcapture %}
+
+{% capture prerequisites %}
+
+1. Three machines running Ubuntu 16.04+, CentOS 7 or HypriotOS v1.0.1+ for the masters. 
```

Comment:
we do support other OSes as well... can you just reference the installing kubeadm guide to be up-to-date?